packages/manager/apps/dedicated/client/app/otrs/otrs.service.js

```diff
@@ -0,0 +1,143 @@
+import identity from 'lodash/identity';
+import pickBy from 'lodash/pickBy';
+import set from 'lodash/set';
+import { Environment } from '@ovh-ux/manager-config';
+
+export default /* @ngInject */ function OtrsOtrsService(
+  $compile,
+  $q,
+  $rootScope,
+  OvhHttp,
+) {
+  const self = this;
+  const cache = {
+    models: 'UNIVERS_MODULE_OTRS_MODELS',
+    me: 'UNIVERS_MODULE_OTRS_ME',
+    vipStatus: 'UNIVERS_MODULE_OTRS_VIP',
+    services: 'UNIVERS_MODULE_OTRS_SERVICES',
+    services2Api: 'UNIVERS_MODULE_OTRS_SERVICES_2API',
+    emails: 'UNIVERS_MODULE_OTRS_EMAILS',
+  };
+
+  this.noAvailableService = [];
+
+  // self.createMenu = Menu.init;
+  self.isLoaded = false;
+
+  self.init = function init() {
+    // self.createMenu();
+    self.isLoaded = true;
+    $compile(angular.element('#otrs-popup'))($rootScope.$new());
```

Comment:
added this line :
the otrs-popup directive code will only be loaded for US region, so we need to compile the directive (https://github.com/ovh/manager/pull/4340/files#diff-4eaf543ac34656361a5e0375c3730f87305031a7a866a005d4ce09d01183723dR134) after the otrs module have been lazy loaded